### ISSUE TITLE
Unhide last activity for admins?

### ISSUE THREAD

**Issue by tvb** [esotalk/esoTalk#372]
Should we unhide the last activity for admins? It might be good to know as an admin who is visiting and reading what on your forum.

**Comment by jgknight** [comment]
I agree, seems that if you're an admin you should be able to see. From the quick look through source, it appears that the hideOnline config key is only used in places that it's displayed -- meaning valid activity data is still stored and could be displayed to the admin user if we choose.
